WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift

```diff
@@ -59,18 +59,27 @@ final class QuickStartPromptViewController: UIViewController {
     private func applyStyles() {
         siteTitleLabel.numberOfLines = 0
         siteTitleLabel.font = WPStyleGuide.fontForTextStyle(.subheadline)
+        siteTitleLabel.adjustsFontForContentSizeCategory = true
+        siteTitleLabel.adjustsFontSizeToFitWidth = true
         siteTitleLabel.textColor = .text
 
         siteDescriptionLabel.numberOfLines = 0
         siteDescriptionLabel.font = WPStyleGuide.fontForTextStyle(.subheadline)
+        siteDescriptionLabel.adjustsFontForContentSizeCategory = true
+        siteDescriptionLabel.adjustsFontSizeToFitWidth = true
         siteDescriptionLabel.textColor = .textSubtle
 
         promptTitleLabel.numberOfLines = 0
         promptTitleLabel.font = WPStyleGuide.fixedSerifFontForTextStyle(.title2, fontWeight: .semibold)
+        promptTitleLabel.adjustsFontForContentSizeCategory = true
+        promptTitleLabel.adjustsFontSizeToFitWidth = true
         promptTitleLabel.textColor = .text
 
         promptDescriptionLabel.numberOfLines = 0
         promptDescriptionLabel.font = WPStyleGuide.fontForTextStyle(.body)
+        promptDescriptionLabel.adjustsFontForContentSizeCategory = true
+        promptDescriptionLabel.adjustsFontSizeToFitWidth = true
+
         promptDescriptionLabel.textColor = .textSubtle
```

Comment:
This is just a very minor, nitpicky note – but the new line is probably not intended here? Considering the `textColor` property belongs to `promptDescriptionLabel`. (Note that this totally is optional and shouldn't block the merge!)

Reply:
> It looks like the updated serif label from #17648 doesn't scale with the font size category.

Ah. You're right :(

@mokagio I'll address the serif scaling in a different PR targeting develop.

> On AX5 size, the text seems to overlap with the action buttons, and I couldn't scroll down. Probably because it's not a scroll view?

I'll embed the text in a scrollview, thanks for pointing that out.

@mokagio Which branch should I target for the scrollview issue? Wondering since the release for 18.8 is very soon.